src/features/providers/lib/utils.ts

```diff
@@ -27,3 +28,23 @@ export function isProviderType(value: unknown): value is ProviderType {
 export function isProviderAuthType(value: unknown): value is ProviderAuthType {
   return Object.values(ProviderAuthType).includes(value as ProviderAuthType);
 }
+
+export function getProviderEndpointByAuthType(provider_type: ProviderType) {
+  return match(provider_type)
+    .with(ProviderType.OPENAI, () => "https://api.openai.com")
+    .with(ProviderType.ANTHROPIC, () => "https://api.anthropic.com")
+    .with(ProviderType.OPENROUTER, () => "https://openrouter.ai")
```

Comment:
```suggestion
    .with(ProviderType.OPENROUTER, () => "https://openrouter.ai/api")
```

I got an error loading models without `/api` here.

Reply:
Thanks 🙏 I will fix it